src/components/Header/Header.js

```diff
@@ -1,8 +1,24 @@
-import React from 'react';
-import { Link } from 'react-router-dom';
+import React, { useEffect, useState } from 'react';
+import { Link, useNavigate } from 'react-router-dom';
 import styled from 'styled-components';
 
 export default function Header() {
+  // 로그인 상태
+  const [isLogin, setIsLogin] = useState(false);
```

Comment:
```suggestion
  const isLogin = localStorage.getItem('token');
```

코드를 보면서 생각해 보니 꼭 state를 사용하지 않고
위와 같은 방법으로 구현해도 되겠습니다!

Reply:
수정 완료하였습니다!